task_11/index.html

```diff
@@ -0,0 +1,389 @@
+                            </label>
+                        </div>
+                        <div class="payMethodItem">
+                            <input name="payItemInput" type="radio" class="visually-hidden" id="paypal" value="paypal" required>
+                            <label class="payBlockLabel" for="paypal">
+                                <img class="payLogo paypalLogo" src="dist/img/pay_pal.png" alt="paypal">
+                            </label>
+                        </div>
+                        <div class="payMethodItem">
+                            <input name="payItemInput" type="radio" class="visually-hidden" id="amazon" value="pamazon" required>
+                            <label class="payBlockLabel" for="amazon">
+                                <img class="payLogo amazonLogo" src="dist/img/amazon.png" alt="amazon">
+                            </label>
+                        </div>
+                    </div>
+                    <div class="inputGroup cardInfoBlock">
+                        <div class="inputItem cardNumber">
+                            <label class="payBlockLabel" for="cardNumber">Card number</label>
+                            <input class="payBlockInput cardNumberInput" type="text" id="cardNumber"
+                                   pattern="\d*" maxlength="16" minlength="16" required>
+                        </div>
+                        <div class="inputItem exDate">
+                            <label class="payBlockLabel" for="exDate">Expiry date</label>
+                            <input class="payBlockInput exDateInput" type="text" id="exDate"
+                                   pattern="^(0[1-9]|1[0-2])\/?([0-9]{4}|[0-9]{2})$" placeholder="DD / MM "
+                                   required>
+                        </div>
+                        <div class="inputItem csv">
+                            <label class="payBlockLabel" for="csv">code</label>
+                            <input class="payBlockInput codeInput" type="text" id="csv" pattern="\d*" minlength="3"
+                                   maxlength="3" required>
+                        </div>
+                    </div>
+                    <div class="termsConditionsBlock">
+                        <input type="checkbox" id="termsAndConditions" required="">
+                        <label class="termsConditionsLabel" for="termsAndConditions">I agree to the
+                            <a href="#">Terms &amp; Conditions</a>
+                        </label>
+                        <button class="button">Place order</button>
+                        <button class="button">Print a qoute</button>
+                    </div>
+                </section>
+            </form>
+        </main>
+    </div>
+    <footer>
+        <div class="wrapperContainer_content footerInfo">
+            <section class="infoFooterSection">
```

Comment:
Header and footer do not enter new section. Usage of section here is not good.

Semantic element for __this case__ is aside because it's additional and mostly unnecessary info.